src/ocean/meta/codegen/CTFE.d

```diff
@@ -30,15 +30,15 @@ import ocean.meta.types.Qualifiers;
 
 *******************************************************************************/
 
-istring toString ( long i )
+istring toString ( T ) ( T i )
```

Comment:
By the way, this is a bit of a weird function / module name for something that only handles integers.

Reply:
Anyone is welcome to implement support for other types if needed :P